qiskit/pulse/__init__.py

```diff
@@ -142,10 +142,7 @@
     Gaussian,
     GaussianSquare,
     GaussianSquareDrag,
-    Sin,
```

Comment:
Seems like a merging error. Those pulses should not be removed.

Reply:
cool, just pushed a fix for this

Reply:
Seems like these are still removed.

Reply:
Now only the `Sin` is still missing.

Reply:
Hopefully they're all back in now